.gitignore

```diff
@@ -1,18 +1,56 @@
 *.gem
 *.rbc
-.bundle
-.config
-coverage
-InstalledFiles
-lib/bundler/man
-pkg
-rdoc
-spec/reports
-test/tmp
-test/version_tmp
-tmp
-
-# YARD artifacts
-.yardoc
-_yardoc
-doc/
+/.config
+/coverage/
+/InstalledFiles
+/pkg/
+/spec/reports/
+/spec/examples.txt
+/test/tmp/
+/test/version_tmp/
+/tmp/
+
+# Used by dotenv library to load environment variables.
+# .env
+
+# Ignore Byebug command history file.
+.byebug_history
+
+## Specific to RubyMotion:
+.dat*
+.repl_history
+build/
+*.bridgesupport
+build-iPhoneOS/
+build-iPhoneSimulator/
+
+## Specific to RubyMotion (use of CocoaPods):
+#
+# We recommend against adding the Pods directory to your .gitignore. However
+# you should judge for yourself, the pros and cons are mentioned at:
+# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
+#
+# vendor/Pods/
+
+## Documentation cache and generated files:
+/.yardoc/
+/_yardoc/
+/doc/
+/rdoc/
+
+## Environment normalization:
+/.bundle/
+/vendor/bundle
+/lib/bundler/man/
+
+# for a library or gem, you might want to ignore these files since the code is
+# intended to run in multiple environments; otherwise, check them in:
+# Gemfile.lock
+# .ruby-version
+# .ruby-gemset
+
```

Comment:
I copied this file from https://github.com/github/gitignore/blob/main/Ruby.gitignore

Should we not be tracking the gemfile in this repo, then?  I'm not familiar with how all this plays together

Reply:
There is a lot of stuff in that gitignore that isn't relevant to us. I'd be hesitant to pull it in in its entirety.

We can probably ignore `Gemfile.lock`, but then we should probably put version constrains in `omniauth-gumroad.gemspec` to match the upgrades we're trying to achieve in the monolith.

Reply:
ie. around here: https://github.com/DripEmail/omniauth-gumroad/blob/master/omniauth-gumroad.gemspec#L19-L20